modules/dr-restoring-cluster-state.adoc

```diff
@@ -459,97 +459,15 @@ clustername-8qw5l-worker-us-east-1c-pkg26   Running   m4.large    us-east-1   us
 ----
 <1> This is the control plane machine for the lost control plane host, `ip-10-0-131-183.ec2.internal`.
 
-.. Save the machine configuration to a file on your file system by running:
-+
-[source,terminal]
-----
-$ oc get machine clustername-8qw5l-master-0 \ <1>
-    -n openshift-machine-api \
-    -o yaml \
-    > new-master-machine.yaml
-----
-<1> Specify the name of the control plane machine for the lost control plane host.
-
-.. Edit the `new-master-machine.yaml` file that was created in the previous step to assign a new name and remove unnecessary fields.
-
-... Remove the entire `status` section by running:
-+
-[source,terminal]
-----
-status:
-  addresses:
-  - address: 10.0.131.183
-    type: InternalIP
-  - address: ip-10-0-131-183.ec2.internal
-    type: InternalDNS
-  - address: ip-10-0-131-183.ec2.internal
-    type: Hostname
-  lastUpdated: "2020-04-20T17:44:29Z"
-  nodeRef:
-    kind: Node
-    name: ip-10-0-131-183.ec2.internal
-    uid: acca4411-af0d-4387-b73e-52b2484295ad
-  phase: Running
-  providerStatus:
-    apiVersion: awsproviderconfig.openshift.io/v1beta1
-    conditions:
-    - lastProbeTime: "2020-04-20T16:53:50Z"
-      lastTransitionTime: "2020-04-20T16:53:50Z"
-      message: machine successfully created
-      reason: MachineCreationSucceeded
-      status: "True"
-      type: MachineCreation
-    instanceId: i-0fdb85790d76d0c3f
-    instanceState: stopped
-    kind: AWSMachineProviderStatus
-----
-
-... Change the `metadata.name` field to a new name by running:
-+
-It is recommended to keep the same base name as the old machine and change the ending number to the next available number. In this example, `clustername-8qw5l-master-0` is changed to `clustername-8qw5l-master-3`:
-+
-[source,terminal]
-----
-apiVersion: machine.openshift.io/v1beta1
-kind: Machine
-metadata:
-  ...
-  name: clustername-8qw5l-master-3
-  ...
-----
-
-... Remove the `spec.providerID` field by running:
-+
-[source,terminal]
-----
-providerID: aws:///us-east-1a/i-0fdb85790d76d0c3f
-----
-
-... Remove the `metadata.annotations` and `metadata.generation` fields by running:
-+
-[source,terminal]
-----
-annotations:
-  machine.openshift.io/instance-state: running
-...
-generation: 2
-----
-
-... Remove the `metadata.resourceVersion` and `metadata.uid` fields by running:
-+
-[source,terminal]
-----
-resourceVersion: "13291"
-uid: a282eb70-40a2-4e89-8009-d05dd420d31a
-----
-
 .. Delete the machine of the lost control plane host by running:
 +
 [source,terminal]
 ----
 $ oc delete machine -n openshift-machine-api clustername-8qw5l-master-0 <1>
 ----
 <1> Specify the name of the control plane machine for the lost control plane host.
++
+A new machine is automatically provisioned after deleting the machine of the lost control plane host.
 
 .. Verify that the machine was deleted by running:
```

Comment:
Since the machine is deleted and automatically provisioned almost immediately, this can be removed in favor of just verifying that the machine is in the provisioning state.